ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -203,6 +190,11 @@ def add_policy(self, name_behavior_id: str, policy: TFPolicy) -> None:
             self.trainer.add_policy(name_behavior_id, policy)
             self._save_snapshot(policy)  # Need to save after trainer initializes policy
             self.learning_behavior_name = name_behavior_id
+            behavior_id_parsed = BehaviorIdentifiers.from_name_behavior_id(
```

Comment:
Should we send the behavior ids as a named tuple to all trainers/ghost trainer instead of parsing this in every advance step?

Reply:
This is being parsed once right? Just at add policy. 

But you're right - maybe we should parse the behavior ID once in the trainer controller rather than inside the trainer, and introduce a new class (BehaviorIdentifier, with properties behavior_id and team_id).

Reply:
The object the parsing returns is a named tuple.  It would probably be good to pass that around instead.  I can make this in a separate PR since I was considering it for multiagent training too.